README.md

```diff
@@ -2,80 +2,24 @@
-<!--TODO: * Finding bugs to work on-->
-<!--TODO: * Getting your code into DevTools' repository (AKA creating and sending patches) -->
-
-### Bugs and issue trackers
-
-Since we have code in two different places, issues and bugs are to be found in two different places:
-
-* For code in `m-c`: [http://firefox-dev.tools/](http://firefox-dev.tools/) which also lets you filter by good bugs for beginners.
-* For code in `devtools-html`: [this page](https://github.com/search?l=&q=org%3Adevtools-html+state%3Aopen&type=Issues) lists all the issues across the organisation and these are [available issues](https://github.com/search?l=&q=org%3Adevtools-html+state%3Aopen+label%3Aavailable&type=Issues) i.e. ready to be worked on.
-
 ## Documentation
 
-Broadly speaking, the tools are divided in two parts: the server and the client. A **server** is anything that can be debugged: for example, your browser, but it could also be Firefox for Android, running on another device. The **client** is the front-end side of the tools, and it is what developers interact with when using the tools.
-
-Since these two parts are decoupled, we can connect to any server using the same client. This enables us to debug multiple types of servers, using the same protocol to communicate.
-
-You will often hear about `actors`. Each feature that can be debugged (for example, network) is exposed via an `actor`, which provides data about that specific feature. It's up to each server to implement some or all actors; the client needs to find out and decide what it can render on the front-side when it connects to the server. So when we want to debug a new feature, we might need to do work in two parts of the code: the server (perhaps implementing a new actor, or extending existing ones) and the client (to display the debugging data returned by the actor).
-
-Often, an actor will correspond to a panel. But a panel might want to get data from multiple actors.
-
-You might also hear about `the toolbox`. The toolbox is what everyone else calls `developer tools` i.e. the front-end that you see when you open the tools in your browser.
-
-This is just a brief overview. For more detailed documentation:
-
-* There is a [docs](https://dxr.mozilla.org/mozilla-central/source/devtools/docs) folder on `m-c` with technical information about the tools and about each individual panel. We are publishing [an HTML version of these documents](http://firefox-dev.tools/docs/) as well.
-* [Internal technical documentation](https://wiki.mozilla.org/DevTools#Internal_Technical_Documentation) (NOTE: we're migrating from here to the folder mentioned above).
-* Additionally, check out each GitHub repository for documentation specific to that repository or module.
+* [End user guides](https://developer.mozilla.org/en-US/docs/Tools).
+* [Developers](https://dxr.mozilla.org/mozilla-central/source/devtools/docs/) documentation - here's [an HTML version of these documents](http://firefox-dev.tools/docs/) as well.
```

Comment:
Let me address this with another updates on another update - will send a patch today